gulp/tasks/e2eTests.js

```diff
@@ -0,0 +1,28 @@
+module.exports = function (gulp, config, plugins) {
+
+    gulp.task('connectDist', function (done) {
+        plugins.connect.server({
+            root: 'dist/',
+            port: 8888
+        });
+    });
+
+    gulp.task('testE2e', function (done) {
```

Comment:
Of course it is possible. I will upload a version with both possible: to do both tests separate and do both task on a single command.